deployments/hadoop-yarn/ansible/38-install-user-db.yml

```
@@ -171,7 +171,7 @@
      copy:
        owner: "{{zepuser}}"
        group: "{{zepuser}}"
-       mode:  'u=rw,g=r,o=r'
+       mode:  '0700'
```

Human readable text is better than numeric.
Useful [calculator tool](https://ss64.com/bash/chmod.html)
```suggestion
       mode:  'u=rwx,g=,o='
```